doc/manual/customizing.rst

```diff
@@ -276,6 +299,25 @@ Other keywords that can be used in a spec:
 * To prevent a control icon from appearing in the Operations plugin
   manager tray specify `optray=False`.
 
+=================
+Disabling Plugins
+=================
+
+Both local and global plugins can be disabled (thus, not shown in the
+reference viewer) using the ``--disable-plugins`` option in the
+command line interface. To remove multiple plugins at once,
+use a comma-separated list. For example::
+
+    ginga --disable-plugins=Zoom,Compose
+
+Alternately, plugins can also be disabled via ``general.cfg`` configuration
+file. For example::
+
+    disable_plugins = "Zoom,Compose"
+
+Some essential plugins, like ``Info``, cannot be disabled (i.e., attempt to
+disable them would have no effect).
```

Comment:
Actually, you can disable `Pan`, `Info`, `Colorbar`, `Toolbar`, `Cursor` and basically any others and Ginga will start up fine (there should be no error messages relating to the missing plugins).  Even `Operations` can be disabled and you can still use Ginga (although some things may not be as convenient).

Reply:
@ejeschke , doesn't work like that for me even using dev. Please see https://github.com/ejeschke/ginga/pull/652#discussion_r194886211

Reply:
I just checked using a fresh instance in a docker container (so downloaded and installed in a clean environment).  I can disable plugins with no problem, and `disable_plugins` works in `general.cfg`.

Can you double check that you don't have an old installation somewhere in your path?  or maybe clear out all the compiled files and try again?

Reply:
I'll follow up tomorrow. Thanks.

Reply:
Sorry, I might be confused by different topics in this thread.  My remarks apply to the "dev" version only.  It's very possible that you can't disable any old plugin in the last "release" version, and the `disable_plugins` feature in `general.cfg` is only available in dev.  But all this will work in 2.7.1.

Reply:
@ejeschke , I was using the latest dev of Ginga but I know why we are both confused now. This is because when "Info" is disabled, I still see the "Info" tab, but I did not realize that "Synopsis" sub-tab was indeed disabled. So, in a way, "Info" is still shown but not functional. :woman_shrugging:

Reply:
(Anyway, I'll remove that blurb from the doc.)

Reply:
Whew!   Yes, that Info tab is part of the workspace, actually, so unless the workspace is altered it will still be created, even though there is no plugin present to populate it.